lib/reporter.js

```diff
@@ -40,6 +40,7 @@ class AllureReporter extends events.EventEmitter {
 
         // Runner events (webdriver)
         this.on('runner:command', ::this.runnerCommand)
+        this.on('runner:customCommand', ::this.customCommand)
```

Comment:
@FelixZilber 
Here and below. let's rename `customCommand` to `step`.
I will be more meaningful

Reply:
what do you think?

Reply:
Agree. Pushed the change